hrm-domain/hrm-service/src/contact/contact-data-access.ts

```diff
@@ -170,23 +191,26 @@ export const create =
     );
   };
 
-export const patch = async (
-  accountSid: string,
-  contactId: string,
-  contactUpdates: ContactUpdates,
-): Promise<Contact | undefined> => {
-  return db.task(async connection => {
-    const updatedContact: Contact = await connection.oneOrNone<Contact>(
-      UPDATE_RAWJSON_BY_ID,
-      {
-        accountSid,
-        contactId,
-        ...contactUpdates,
-      },
-    );
-    return updatedContact;
-  });
-};
+export const patch =
+  (task?) =>
+  async (
+    accountSid: string,
+    contactId: string,
+    contactUpdates: ContactUpdates,
+  ): Promise<Contact | undefined> => {
+    return txIfNotInOne(task, async connection => {
+      const updatedContact: Contact = await connection.oneOrNone<Contact>(
+        UPDATE_CONTACT_BY_ID,
+        {
+          ...BLANK_CONTACT_UPDATES,
+          accountSid,
+          contactId,
+          ...contactUpdates,
```

Comment:
Won't `BLANK_CONTACT_UPDATES` set any column missing in `contactUpdates` to be set as null in the table?